### PR TITLE
feat: Adds Gitpod configuration

### PR DESCRIPTION
**Pull request by akoenig**
This PR introduces our Gitpod setup.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -4,5 +4,11 @@
     "deno.unstable": true,
     "deno.suggest.imports.hosts": {
         "https://deno.land": true
-    }
+    },
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.formatOnSave": true,
+    "workbench.colorTheme": "GitHub Dark",
+    "git.autofetch": true,
+    "gitlens.showWelcomeOnInstall": false,
```
Comment (grebaldi) [inline .vscode/settings.json:12]:
```suggestion
```

These three options shouldn't be here, because they override user defaults for everyone.

**Review by grebaldi — CHANGES_REQUESTED** [review]
Hi @akoenig,

thanks for taking care!

I was able to verify that the setup works, but added a comment regarding `.vscode/settings.json`.

File: .vscode/settings.json (hunk 1 above)
Comment (akoenig) [reply .vscode/settings.json:12]: Hey @grebaldi, thanks for the review 🙂 I agree that `workbench.colorTheme` and the `git.autofetch` is definitely really opinionated and we _could_ remove it.

Regarding `gitlens.showWelcomeOnInstall`: I think we should deactivate the welcome page as GitLens gets installed through the extension configuration in the `.gitpod.yml`. So when you throw away a Gitpod workspace and start a new one (e.g. when reviewing a PR), you will see the welcome page again. IMHO this gets annoying pretty quickly 🙂

Comment (grebaldi) [reply .vscode/settings.json:12]: Sure, I don't think anybody would like to keep that setting on `true` anyway, so it's not that intrusive :smile:

**Comment by valarot** [comment]
@akoenig piep

**Comment by akoenig** [comment]
Hey @grebaldi, removed the opinionated VS Code settings 🙂

**Review by grebaldi — APPROVED** [review]

**Pull request merged**